lib/versioning/maven/compare.ts

```diff
@@ -545,13 +541,15 @@ function autoExtendMavenRange(
     interval.leftValue = coerceRangeValue(leftValue, newValue);
   }
 
-  if (interval.leftValue && interval.rightValue) {
-    const correctRepresentation =
-      compare(interval.leftValue, interval.rightValue) === 1
-        ? null
-        : rangeToStr(range);
-    return correctRepresentation || currentRepresentation;
+  // istanbul ignore if
```

Comment:
No test for this?

Reply:
Seems like we don't need this check anymore. But I'm not sure , so let it will be kind of *soft assert*. Log a warning, maybe?

Reply:
sure, can you create a synthetic test for it? if not possible we should remove that code if it's not possible to get there.

Reply:
Okay, I can't create test, guess we've fixed it 😎